src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtCaseEventsListenerIntTest.kt

```diff
@@ -167,11 +166,19 @@ class CourtCaseEventsListenerIntTest : IntegrationTestBase() {
         },
       )
     }
+
+    await untilAsserted {
+      verify(telemetryService).trackEvent(
+        eq(TelemetryEventType.INVALID_PNC),
+        check {
+          assertThat(it["PNC]"]).isNullOrEmpty()
+        },
+      )
+    }
   }
 
   @Test
-  @Disabled("Works locally - disabled until reason for failure on circleCi has been established")
```

Comment:
Great to see this has gone - we should also enable or delete all the other disabled tests.